apps/new-dealer/models/submission.js

```diff
@@ -2,10 +2,27 @@
 /* eslint complexity: 0 max-statements: 0 */
 const contains = (arr, val) => arr.indexOf(val) > -1 ? 'Yes' : 'No';
 
+const authorityType = usage => {
+  if (usage.indexOf('arm-guards') > -1) {
+    return 'Maritime Guards';
+  }
+
+  if (usage.indexOf('transport') > -1 || usage.indexOf('transfer') > -1) {
```

Comment:
I think you're right, going to look somewhere else to decipher the logic would not be a better idea. But perhaps `lodash#_.includes` might improve the readability. I tend to find several instances of `indexOf ... > -1` difficult to grok at a glance. That said, I am not super precious about this, just want to keep it simple.